### ISSUE TITLE
signals with multiple parameters

### ISSUE THREAD

**Issue by adriantofan** [ReactiveCocoa/ReactiveCocoa#449]
Hello,

First of all thank you for writing this library!

I would like to ask your help for a design problem. I am working with an XMPP / Jabber library which is implemented as a huge state machine. 

I have defined some callbacks, where I generate signals, which are used later in the UI code. 

For example:

``` objective-c
self.signalCallStateChange = [RACSubject subject];

-(void)callStateChanged:(int)state jid:(char*)jid callId:(int)callId{
  [self.signalCallStateChange sendNext:....];
}
```

My question is about sending multiple items in a signal such as in this case.

What I have done is to define a container class for the state change. (I really don't want to put everything inside a NSDictionary)

``` objective-c
@interface StateChange : NSObject
@property (nonatomic, assign) NSInteger state;
@property (nonatomic, strong) NSString* jid;
@property (nonatomic, assign) NSInteger callID;
@end
```

and later on:

``` objective-c
-(void)callStateChanged:(int)state jid:(char*)jid callId:(int)callId{
  StateChange* chage = [StateChange new];
  chage.state = state;
  chage.jid = [NSString stringWithUTF8String:jid];
  chage.callID = callId;
  [self.signalCallStateChange sendNext:chage];
}
```

Basically for each multiple argument callback declared by the library I need to create a container object in order to pass it to the signal.  Even more , I need to document somehow for each public signal what kind of object to expect.

Do you see a better way of doing this?
